manager/testdata/notebook/read-flow/pii-policy-cm.yaml

```diff
@@ -3,17 +3,12 @@ data:
   policy-read.rego: |+
     package dataapi.authz
 
-    rule[{}] {
-        description := "allow read datasets with no tags"
-        input.action.actionType == "read"
-        not input.resource.metadata.tags
-    }
-
     rule[{"action": {"name":"RedactAction", "columns": column_names}, "policy": description}] {
-        description := "While reading redact columns tagged as PII in datasets tagged with finance = true"
+        description := "Redact columns tagged as PII.Sensitive in datasets tagged with Purpose.finance = true"
         input.action.actionType == "read"
-        input.resource.metadata.tags.finance
-        column_names := [input.resource.metadata.columns[i].name | input.resource.metadata.columns[i].tags.PII]
+        input.resource.metadata.tags["Purpose.finance"]
+        column_names := [input.resource.metadata.columns[i].name | input.resource.metadata.columns[i].tags["PII.Sensitive"]]
+        count(column_names) > 0
```

Comment:
I compare these roles with the roles in the write scenario, both Web site and the test.
The write policy looks like :

```rego
rule[{}] {
        description := "allow read datasets"
        input.action.actionType == "read"
    }

    rule[{"action": {"name":"RedactAction","columns": column_names}, "policy": description}] {
        description := "Redact written columns tagged as sensitive in datasets tagged with finance = true. The data should not be stored in `neverland` storage account"
        input.action.actionType == "write"
        input.resource.metadata.tags.finance
        input.action.destination != "neverland"
        column_names := [input.resource.metadata.columns[i].name | input.resource.metadata.columns[i].tags.sensitive]
    }
```

Maybe we have to sync all examples and tests.

Reply:
yes, as noted in https://github.com/fybrik/fybrik/pull/1994 the notebook write test still differs from the one in the site due to:

- the default policy is deny instead of allow in the site
- adminCRsNamespace is set to default namespace instead of fybrik-system in the site.

this was done to reduce the number of tests.